test/scripts/e2e_subs/create_destroy_optin_optout.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script creates some state required for Indexer testing
+# TIMEOUT=400
+
+date '+create_destroy_optin_optout start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+# Directory of this bash program
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list | awk '{ print $3 }')
+ACCOUNTB=$(${gcmd} account new | awk '{ print $6 }')
+${gcmd} clerk send -a 1000000 -f $ACCOUNT -t $ACCOUNTB
+
+#############################################
+# Account - create / close / create / close #
+#############################################
+ACCOUNT_NEW=$(${gcmd} account new | grep "Created" | awk '{ sub("\r", "", $NF); print $NF }')
```

Comment:
is there any particular reason you're going this differently as the above ACCOUNTB ?
( I'm not particular, but would rather be consistent )

Reply:
I also suspect ACCOUNT_NEW would look better as ACCOUNTC